spec/features/journeys/basic_registration_journey_spec.rb

```diff
@@ -200,7 +195,7 @@
         "funding" => "trust",
         "institution_identifier" => "School-100000",
         "institution_location" => "manchester",
-        "institution_name" => "",
+        "institution_name" => js ? "" : "open", # FIXME: i have no idea why this is different
```

Comment:
We appear to write an extra attribute when picking a school without JS, I guess we submit a search rather than instigating a JS autocompletion.

Reply:
The comment is correct - institution_name is populated via a no-js fallback search